python/dolfin/__init__.py

```diff
@@ -40,8 +40,7 @@
 
 import dolfin.MPI
 
-from .cpp.function import (Expression, Constant)
-from .cpp.fem import (FiniteElement, DofMap)
+from dolfin.fem import DofMap
```

Comment:
Do we want this at the top level?

Reply:
Now I am just working within `python/dolfin/` keeping global imports untouched (i.e. user space). 
After everything is wrapped I'll remove global imports and this needs a lot of work in tests/demos. Otherwise this PR will loose its 'locality'.

Reply:
Ok, leave it for now. I think we'll remove it pretty soon.